docs/topics/auth/customizing.txt

```diff
@@ -255,15 +269,56 @@ authenticated users do not.
 Do not forget to test for the ``is_active`` attribute of the user in your own
 backend permission methods.
 
+.. _object_permissions:
+
 Handling object permissions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Django's permission framework has a foundation for object permissions, though
-there is no implementation for it in the core. That means that checking for
-object permissions will always return ``False`` or an empty list (depending on
-the check performed). An authentication backend will receive the keyword
-parameters ``obj`` and ``user_obj`` for each object related authorization
-method and can return the object level permission as appropriate.
+there is no implementation for it in the core. An authentication backend will
```

Comment:
With the combined changes here this line is no longer true. There **is** an implementation in core.

Reply:
I think that would be an overstatement. `DefaultObjectPermissionObject` is not useful on its own. If you use only `ModelBackend` and `DefaultObjectPermissionObject` you could just as well only use the former and not pass objects.

`DefaultObjectPermissionObject` is interesting if you use one backend handeling global permissions and a second one handeling object permissions.

Reply:
OK, leave this one.